CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## 2.1.0 (October 20, 2015)
+### general
+ - Added ability to install and upgrade Logstash plugins without requiring internet connectivity (#2376). 
+ - Support alternate or private Ruby gems server to install and update plugins (#3576).
+ - Fixed a memory leak which could be triggered when events having a date were serialized to string (#4222).
+ - Added JDBC input to default package.
+ - Logstash can read multiple config files from a directory. If you combine --config_test CLI flag with 
+   `--debug`, Logstash will log the combined config file, annotating the individual config blocks
+   with the source file it came from (#3243)
+ - Reset default worker threads to 1 when using non thread-safe filter like multiline (#4130).
+ - Fixed file permissions for the logrotate configuration file.
+ - Changed the default heap size from 500MB to 1GB (#3861)
+ - Fixed config check option when starting Logstash through init scripts (#3645)
+
+ ### input
```

Comment:
this section is indented one level, which makes input/output sections misaligned with general